engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -540,6 +540,16 @@ void Canvas::DrawRoundRect(const RoundRect& round_rect, const Paint& paint) {
   DrawPath(path, paint);
 }
 
+void Canvas::DrawRoundSuperellipse(const RoundSuperellipse& rse,
+                                   const Paint& paint) {
+  Entity entity;
+  entity.SetTransform(GetCurrentTransform());
+  entity.SetBlendMode(paint.blend_mode);
+
+  RoundSuperellipseGeometry geom(rse.GetBounds(), rse.GetRadii());
```

Comment:
We could probably attempt_blurred_rrect if the mask blur is high enough?

Reply:
Yeah that's what I thought as well, but it's a bit risky and I don't think I should implement it without sufficient investigation.